tests/local-values.yml

```diff
@@ -0,0 +1,75 @@
+nodeCount: &nodeCount 1
+
+ingress:
+  enabled: true
+  hosts:
+    - ""
+
+prometheus:
+  rbac:
+    create: false
+  serviceAccounts:
+    alertmanager:
+      name: default
+      create: false
+    nodeExporter:
+      name: default
+      create: false
+    kubeStateMetrics:
+      name: default
+      create: false
+    pushgateway:
+      name: default
+      create: false
+    server:
+      name: default
+      create: false
+
+redis-ha:
+  rbac:
+    create: false
+  serviceAccount:
+    create: false
+    name: default
+  replicas:
+    servers: *nodeCount
+    sentinels: *nodeCount
+
+cloudserver-front:
+  image:
+    pullPolicy: Always
+  orbit:
+    endpoint: "http://ciutil-orbit-simulator:4222"
```

Comment:
If this is for local testing do we want orbit-simulator references?

Reply:
orbit sim still runs for local test runs